ome_zarr/writer.py

```diff
@@ -202,15 +203,27 @@ def write_multiscale(
       For each path, we have a List of transformation Dicts.
       Each list of dicts are added to each datasets in order
       and must include a 'scale' transform.
+    storage_options: dict or list of dict
+      Options to be passed on to the storage backend. A list would need to match
+      the number of datasets in a multiresolution pyramid. One can provide
+      different chunk size for each level of a pyramind using this option.
     """
 
     dims = len(pyramid[0].shape)
     axes = _get_valid_axes(dims, axes, fmt)
 
     datasets: List[dict] = []
     for path, data in enumerate(pyramid):
-        # TODO: chunks here could be different per layer
-        group.create_dataset(str(path), data=data, chunks=chunks)
+        options = {}
+        if storage_options:
+            options = (
+                storage_options
+                if not isinstance(storage_options, list)
+                else storage_options[path]
+            )
+        if "chunks" not in options:
+            options["chunks"] = chunks
```

Comment:
This raises the question of whether we would like to deprecate the explicit `chunks` argument and only rely on the `storage_options` dictionary moving forward.

Reply:
if it's deprecated, i would suggest giving it a couple of cycles and perhaps add a deprecation warning with the next release. but leave this up to you.

Reply:
Fair enough, we can gain knowledge about the `storage_options` in the upcoming 0.3.x and schedule the deprecation of `chunks` for 0.4. I'll create an issue